component/exoscale_postgres.jsonnet

```diff
@@ -71,17 +71,11 @@ local composition =
     spec: {
       compositeTypeRef: comp.CompositeRef(xrd),
       writeConnectionSecretsToNamespace: pgParams.secretNamespace,
-      patchSets: [
-        comp.PatchSet('annotations'),
-        comp.PatchSet('labels'),
-      ],
       resources: [
         {
           base: pgBase,
           connectionDetails: comp.conn.AllFromSecretKeys(connectionSecretKeys),
           patches: [
-            comp.PatchSetRef('annotations'),
-            comp.PatchSetRef('labels'),
             comp.FromCompositeFieldPath('metadata.labels[crossplane.io/composite]', 'metadata.name'),
```

Comment:
AFAIK the billing collector depends on the label `crossplane.io/claim-namespace` being set. See: https://github.com/vshn/billing-collector-cloudservices/blob/master/pkg/exoscale/dbaas.go#L175

We'll need to explicitly forward that. Otherwise I think this should be fine

Reply:
The `crossplane.io/*` labels are autogenerated. The only thing that the removed patchsets forward are user set annotations and labels on the claim. I'm not really sure why we did this in the first place...

After creating an `evil` intance without the patchsets there are still the labels on the namespaces:

```
apiVersion: v1
kind: Namespace
metadata:
  labels:
    appcat.vshn.io/claim-namespace: default
    appcat.vshn.io/servicename: postgresql-standalone
    kubernetes.io/metadata.name: vshn-postgresql-evil-m89lq
  name: vshn-postgresql-evil-m89lq
```

Reply:
Looks like those labels are autopopulated to managed resources as well:

```
k get postgresqls.exoscale.crossplane.io my-postgres-example-27898 -o jsonpath='{.metadata.labels}' | jq
{
  "crossplane.io/claim-name": "my-postgres-example",
  "crossplane.io/claim-namespace": "default",
  "crossplane.io/composite": "my-postgres-example-27898"
}
```

There's no need to explicitly patch them.